ext/grpc/php_grpc.c

```diff
@@ -230,9 +230,6 @@ PHP_MINIT_FUNCTION(grpc) {
   REGISTER_LONG_CONSTANT("Grpc\\CHANNEL_TRANSIENT_FAILURE",
                          GRPC_CHANNEL_TRANSIENT_FAILURE,
                          CONST_CS | CONST_PERSISTENT);
-  REGISTER_LONG_CONSTANT("Grpc\\CHANNEL_FATAL_FAILURE",
-                         GRPC_CHANNEL_FATAL_FAILURE,
```

Comment:
Not should delete the three codes, you shoud update `GRPC_CHANNEL_FATAL_FAILURE` to `GRPC_CHANNEL_SHOWDOWN`.